applications/nrf5340_audio/src/bluetooth/Kconfig.defaults

```diff
@@ -86,8 +86,7 @@ config BT_ASCS_ASE_SNK_COUNT
 	default 1
 
 config BT_ASCS_ASE_SRC_COUNT
-	default 1 if STREAM_BIDIRECTIONAL
-	default 0
+	default 1
```

Comment:
Why?

Reply:
Due to a bug in host, fixed in https://github.com/zephyrproject-rtos/zephyr/pull/52026

Reply:
If I cherry-pick the fix as `[nrf fromtree]` then this change can be removed?

Reply:
yes :)

Reply:
Both commits from the fix are now part of upmerge. Please update this PR.